### ISSUE TITLE
Story for multi-language sites

### ISSUE THREAD

**Issue by saumier** [culturecreates/footlight-app#1371]
As a contributor to a multi-lingual website, like NAC Osaka, I specialize in one language (Japanese) and collaborate with my colleagues who each specialize in their language.  

When I work on people, places and organizations to translate them into the language I specialize in, I need to see which entities are not translated, and work on those in parallel with my colleagues.

When not editing, I would like to display labels of entities in lists, like the list of peoples names, in my language (content language) EVEN if my language is not one of the UI languages.

I would like to have the read-only page default to my content language, so I don't always have to switch.  
When I open the edit page, the tabs should default to my content language so I don't always have to click them to switch.

When editing events, I may have to edit them after they have been published.  If a last minute show is added or changed, it will already be published before I have a chance to work on the translation.

In general, my work is ongoing and can happen before, during or after events happen.

**Comment by saumier** [comment]
@troughc Please take a look.

**Comment by saumier** [comment]
@troughc I modified the user story so it does not contain solutions, and I added a section outside of the user story with ideas for solutions of which some or none can be chose for implementation.

**Comment by troughc** [comment]
Thanks @saumier. As discussed, I am going to see about retaining the language literal as is until someone explicitly edits/updates the content in another language (inherited content from another language). That way every user that views an edit form after saving can be more aware.

**Comment by saumier** [comment]
@troughc Yes. Based on the hypothetical user story, wouldn't it be useful for a translator to have a way to list the entities missing a translation?
